_includes/scenarios/integrate/prepare-enmeshed-onboarding-package.md

```diff
@@ -0,0 +1,132 @@
+
+<!---{% include rapidoc api_route_regex="^post /api/v2/RelationshipTemplates/Own$" %}--->
+
+### Success Response
+
+If you have successfully created the RelationshipTemplate on your Connector on a certain Device, you will receive a success response in the following form:
+
+```jsonc
+{
+  "result": {
+    "id": "<your RelationshipTemplate's ID>",
+    "isOwn": true,
+    "createdBy": "<your Connector's ID>",
+    "createdByDevice": "<your Device's ID>",
+    "createdAt": "<creation date>",
+    "expiresAt": "<expiration date>",
+    "content": {
+      //Content of the RelationshipTemplate
+      ...
+    },
+    "truncatedReference": "<your RelationshipTemplate's truncated reference>",
+    "maxNumberOfAllocations": <maximum number of allocations>,
+    "secretKey": "<your RelationshipTemplate's secret key>"
+  }
+}
+```
+
+{% include copy-notice description="The ID of the RelationshipTemplate can be read from the `id` property. Save it so that you can refer to your RelationshipTemplate later. For the same reason, save the values of the `truncatedReference` and `secretKey` properties." %}
+
+## Onboarding
+
+Before an Identity can establish a Relationship with your Connector, it must send a Relationship Request specifying the ID of a valid RelationshipTemplate owned by your Connector. Depending on whether the Identity is an App user or another Connector, a different approach must be used to make the RelationshipTemplate available to it:
+
+- Onboarding of an App user: Scan the QR Code of the RelationshipTemplate.
+- Onboarding of another Connector: Load the RelationshipTemplate onto it.
+
+[![High level architecture diagram of enmeshed components and layers]({{ '/assets/images/integrate/Prepare onboarding package.svg' | relative_url }}){: .align-center}]({{ '/assets/images/integrate/Prepare onboarding package.svg' | relative_url }})
+
+<!--- Not magnifiable version: ![High level architecture diagram of enmeshed components and layers]({{ '/assets/images/integrate/Prepare enmeshed onboarding package.svg' | relative_url }}){: .align-center} --->
+
+### Onboarding of an App user
+
+If an App user wants to send a Relationship Request to your Connector, the App user must first scan a QR Code that contains the reference to a RelationshipTemplate owned by your Connector. By sending the Request `GET /api/v2/RelationshipTemplates/<id>`, specifying the value `image/png` in the `Accept` header field, you can create this QR Code on your Connector. You must replace the placeholder `<id>` in the URL with the ID of the RelationshipTemplate obtained from the [success response]({% link _docs_integrate/prepare-enmeshed-onboarding-package.md %}#success-response) above.
+
+For more details on how to send a Request to create a QR Code containing the reference to a RelationshipTemplate, see the description of the [Get RelationshipTemplate]({% link _docs_use-cases/use-case-transport-get-relationship-template.md %}) usecase.
+{: .notice--info}
```

Comment:
Maybe wait until the usecase isn't empty anymore, before linking it (also below).

Reply:
In some cases my documentation is depending on the use cases, so maybe we need to wait for the required use cases to be reviewed and published